lib/loaders/config-loader/index.js

```diff
@@ -23,7 +23,6 @@
     value.relativePath = path.relative('.', directory);
     return globPages(directory, function(err, pagesData) {
       value.pages = pagesData;
-      this.value = [value];
```

Comment:
Yeah, forgot I bumped my head over it a while, might be some transformation that fails.
I'm not used to CoffeeScript now understand that @ notation is the this scope notation (right?), but while executing with that line this error pops up:
`Cannot set property 'value' of undefined`

For what I see the "value" is just passed in "callback" (wrapped as an array?).